### PR TITLE
Viper 15: Create HeaderUI

### PR DESCRIPTION
**Pull request by hthh03**
Complete the basic code to create header UI

### CODE REVIEW

File: Viper-15/UI.css
Comment (thienbuidev) [inline Viper-15/UI.css:1]: name the file to match the html file name to differentiate

Comment (thienbuidev) [reply Viper-15/UI.css:1]: Change folder with your name

File: Viper-15/headerUI.html
Comment (thienbuidev) [inline Viper-15/headerUI.html:1]: folder contains only css and js

**Review by thienbuidev — CHANGES_REQUESTED** [review]

File: Lab-01/index.html
Comment (bxthien) [inline Lab-01/index.html:1]: this file no need in project. pls delete

**Review by bxthien — CHANGES_REQUESTED** [review]

**Review by bxthien — APPROVED** [review]

**Pull request merged**